docker-connector/src/com/axonivy/connector/docker/IvyDockerClient.java

```diff
@@ -0,0 +1,117 @@
+package com.axonivy.connector.docker;
+
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import com.github.dockerjava.api.command.ListContainersCmd;
+import com.github.dockerjava.api.model.Network;
+import com.github.dockerjava.core.DefaultDockerClientConfig;
+import com.github.dockerjava.core.DockerClientImpl;
+import com.github.dockerjava.httpclient5.ApacheDockerHttpClient;
+
+import ch.ivyteam.ivy.environment.Ivy;
+
+public class IvyDockerClient {
+
+  private static final IvyDockerClient INSTANCE = new IvyDockerClient();
+  private DockerClient client;
+
+  private IvyDockerClient() {
+    client = createDockerClient();
+  }
+
+  public static IvyDockerClient instance() {
+    return INSTANCE;
+  }
+
+  private static DockerClient createDockerClient() {
+    var dockerUrl = Ivy.var().get("Docker.URL");
+    var config = DefaultDockerClientConfig.createDefaultConfigBuilder()
+            .withDockerHost(dockerUrl)
+//            .withDockerTlsVerify(true)
```

Comment:
Please remove this comment if it is not necessary, or leave a comment explaining why we need it.